code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -9,8 +9,8 @@
 	var/labelled = FALSE
 	fill_icon_thresholds = list(10, 20, 30, 40, 50, 60, 70, 80, 90, 100)
 
-/obj/item/reagent_containers/blood/Initialize(mapload)
-	. = ..()
+/obj/item/reagent_containers/blood/Initialize(mapload, vol)
+	. = ..(mapload, vol)
```

Comment:
wait why are we doing it like this, you don't mutate the args from what's passed into initialize (actually, why does `vol` need to be an arg on initialize)- shouldn't need to modify the args like this